README.md

```diff
@@ -90,3 +90,16 @@ For Payment Hub usecases, check this [documentation](https://mifos.gitbook.io/do
 ## Note
 
 The UI design is currently being revamped. New design can be found [here](https://docs.google.com/presentation/d/1yFR19vGlKW-amxzGms8TgPzd1jWkrALPFcaC85EyYpw/edit#slide=id.g6c6ccd991d_0_42)
+
+## Join Us on Slack
+
+Mifos boasts an active and vibrant contributor community, which you can engage with [here](https://mifos.slack.com/). Once you've signed in, please explore the various channels, and make sure to join the `#mifos-mobile` channel. This channel encompasses all members involved with this specific project.
```

Comment:
Please update the text with 

Mifos boasts an active and vibrant contributor community, Please join us on slack https://mifos.slack.com/ Once you've joined the mifos slack community, please join the `#mifos-mobile` channel to engage with mifos-mobile development.

Reply:
done